tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from django.template import engines
+
+
+@pytest.fixture
+def rusty():
+    return engines["rusty"].from_string
+
+
+@pytest.fixture
+def django_temp():
+    return engines["django"].from_string
+
+
+@pytest.fixture
+def assert_render(rusty, django_temp):
+    def temp(template, context, expected):
+        assert django_temp(template).render(context) == expected
+        assert rusty(template).render(context) == expected
+
+    return temp
```

Comment:
I found myself reading `temp` as "temporary" rather than "template":

```suggestion
    def render_template(template, context, expected):
        assert django_temp(template).render(context) == expected
        assert rusty(template).render(context) == expected

    return render_template
```